lib/vanagon/component/dsl.rb

```diff
@@ -184,7 +184,8 @@ def install_service(service_file, default_file = nil, service_name = @component.
           # Return here because there is no file to install, just a string read in
           return
         when "windows"
-          # Placeholder for Windows
+          @component.service = OpenStruct.new(:name => service_name, :service_file => service_file)
```

Comment:
```:name``` isn't required, but I thought it a useful key to leave in there for debugging, etc. etc.
I can add that detail in as a comment here, but wondering if its better to put it elsewhere.
You can also take a look at PR https://github.com/puppetlabs/puppet-agent/pull/514/files

Are you thinking of a comment like:
```# pkg.install_service "SourceDir\\ProgramFiles64Folder\\PuppetLabs\\Puppet\\bin\\ruby.exe"```
and a note that a separate service wix file also needs to be provided.